ch05construction/02conventions.ipynb

```diff
@@ -240,8 +254,27 @@
    },
    "outputs": [],
    "source": [
-    "fNumber= float(sInput) + iOffset\n",
-    "number = float(input)  + offset"
+    "fNumber = float(sEntry) + iOffset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So in the example above we know that we are creating a `f`loat number as a composition of a `s`tring entry and a `i`nteger offset.\n",
+    "\n",
+    "People may find this useful on languages like Python where the type is intrisic in the variable."
```

Comment:
I would say "useful in languages like Python", and I'm not sure whether "intrinsic in the variable" is totally clear. Does it mean that the type is not given explicitly?

Reply:
As the type is not defined explicitly.